packages/datadog-instrumentations/src/openai.js

```diff
@@ -3,8 +3,12 @@
 const { addHook } = require('./helpers/instrument')
 const shimmer = require('../../datadog-shimmer')
 
-const tracingChannel = require('dc-polyfill').tracingChannel
-const ch = tracingChannel('apm:openai:request')
+const dc = require('dc-polyfill')
+const ch = dc.tracingChannel('apm:openai:request')
+
+// this will tell the span to finish
+// this is so all subscribers/plugins can use the span before it is finished/processed
+const finishCh = dc.channel('tracing:apm:openai:request:finish')
```

Comment:
APM reviewers: see description for why I added this. tl;dr, I want to separate _tagging_ at the `asyncEnd` lifecycle event from the actual span finishing.

Reply:
If needed, I can revert this and move forward for now (by putting the LLMObs plugin before the tracing plugin so its `asyncEnd` runs first), and maybe we can find a better solution down the road...

Reply:
I've actually removed this, and updated the PR description